Replace or remove update with replace

tkrille opened the pull request:
OSIAM 3.0 removes support for updates via PATCH, so we have to test the
PUT-based API from now on and remove the tests for the PATCH-based one.

Depends on osiam/connector4java#210

tkrille commented:
Depends on https://github.com/osiam/connector4java-integration-tests/pull/343

wallner commented on src/test/groovy/org/osiam/test/integration/ControllerIT.groovy:
```diff
@@ -319,34 +318,21 @@ class ControllerIT extends AbstractIT {
-        UpdateUser updateUser = new UpdateUser.Builder().updateActive(false).build()
+        def serviceAccessToken = OSIAM_CONNECTOR.retrieveAccessToken(Scope.ADMIN)
+        def user = OSIAM_CONNECTOR.getUser(userId, serviceAccessToken)
+        def updateUser = new User.Builder(user).setActive(false).build()
```
Please move the initialization down into the 'when' block. It makes it easier to understand.

tkrille replied:
Done

wallner commented on src/test/groovy/org/osiam/test/integration/UserGroupMembershipIT.groovy:
```diff
@@ -46,12 +46,12 @@ class UserGroupMembershipIT extends AbstractIT {
 
-        def updateGroup = new UpdateGroup.Builder()
-                .addMember(createdUser.getId())
+        def updateGroup = new Group.Builder(createdGroup)
```
I would suggest movingthe initialization into the 'when' block.

tkrille replied:
Done

tkrille commented:
Updated

wallner commented on src/test/groovy/org/osiam/test/integration/UserGroupMembershipIT.groovy:
```diff
@@ -104,12 +103,12 @@ class UserGroupMembershipIT extends AbstractIT {
 
-        UpdateGroup updateGroup = new UpdateGroup.Builder()
-                .deleteMember(memberGroup1.getId())
+        def updateGroup = new Group.Builder(retParentGroup)
```
Don't you thinkthis belongs to the 'when' block too?

wallner commented on src/test/groovy/org/osiam/test/integration/UserGroupMembershipIT.groovy:
```diff
@@ -132,12 +131,12 @@ class UserGroupMembershipIT extends AbstractIT {
 
-        UpdateGroup updateGroup = new UpdateGroup.Builder()
-                .deleteMembers()
+        def updateGroup = new Group.Builder(parent)
```
Please move the initialization to the 'when' block.

tkrille commented:
Alright, @wallner, I've just replaced PATCH with PUT and did not touch any other things. I don't think it's viable that I improve all tests I've changed now. I will fix your last 2 comments, but that should be the last improvements.

tkrille commented:
Updated

wallner commented:
Ok, thank you!
